src/engine/strat_engine/filesystem.rs

```diff
@@ -64,6 +97,33 @@ impl StratFilesystem {
         self.thin_dev.id()
     }
 
+    /// Return an target size for the filesystem over the thin device.
+    /// TODO: returning 2 times the current size will double the space provisoned to
+    /// the filesystem.  We should determine if this is a reasonable value.
+    fn extend_size(&self, current_size: DataBlocks) -> DataBlocks {
+        // xfs_growfs takes the target size as a parameter
```

Comment:
Please get rid of comment and instead change name of function OR keep the name of the function, have it do the same thing as the other ```extend_size()``` function, and change how you use the value returned at the call site.